Adj lpu and stock view

eHanseFrank opened the pull request:
- is this Pull request okay? Want to merge my adj_lpu_and_stock_view branche into your add_rmb_rate branch - sorry my cluelessness

- trying badly to get the image_small from product_template via ENV but it is not working somehow.

eHanseFrank committed:
Date instead of Datetime

schumach86 committed:
IMP on List Price Update

yostashiro committed:
[REF] misc corrections

yostashiro committed:
[IMP] show RMB rate

eHanseFrank committed:
Change Order Of Attributes

eHanseFrank committed:
Merges attribute order with cny_rate

eHanseFrank committed:
Using cny rate to calculate net_price_rnb and further changes to attribute order in LPU

eHanseFrank committed:
Including corrections on cny price and added images to stock_view

eHanseFrank committed:
Trying to get products_template's small image into stock.quant via ENV

yostashiro commented:
@eHanseFrank You may rebase your branch to `add-rmb-rate` branch so that there will be no past commits included in the PR.

Regarding getting `the image_small`, does `related='product_id.product_tmpl_id.image_small'` not work?

yostashiro commented:
@eHanseFrank Do you think I should go ahead and merge `add-rmb-rate` branch to `rfhk/master` for now?  You can create your PR to `rfhk/master` later on.  What do you think?

eHanseFrank commented:
Hi Yoshi,


i reviewed the code and i see that some modification I suggested last 
week have not been included yet. So i created a new branch, rebased it 
to rfhk/master and your fork (the latter rebase one should have been 
sufficient, right?)

added again the minor modifications and created a pull request. Please 
let me know,

- wheter my pull request is still messy

- wheter those changes regarding rmb_net_price are okay


Thanks

Frank


Am 30-03-2017 um 8:04 PM schrieb Yoshi Tashiro:
>
> @eHanseFrank <https://github.com/eHanseFrank> Do you think I should go 
> ahead and merge |add-rmb-rate| branch to |rfhk/master| for now? You 
> can create your PR to |rfhk/master| later on. What do you think?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/yostashiro/awo-custom/pull/6#issuecomment-290390562>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AKqR6lF3OG-sXJyI8BPFjoIUiy7xB5H2ks5rq5pOgaJpZM4MiRe7>.
>
